vtr_flow/tasks/regression_tests/vtr_reg_strong/task_list.txt

```diff
@@ -70,3 +70,7 @@ regression_tests/vtr_reg_strong/strong_timing_update_type
 regression_tests/vtr_reg_strong/strong_timing_update_diff
 regression_tests/vtr_reg_strong/strong_blocks_with_no_inputs
 regression_tests/vtr_reg_strong/strong_fix_clusters
+regression_tests/vtr_reg_strong/strong_analytic_placer
+regression_tests/vtr_reg_strong/strong_timing_update_diff
```

Comment:
It looks like some tests got duplicated (timing_update_diff, string_fix_clusters)?